includes/core/server/request/class-get-reporting-payment-activity.php

```diff
@@ -50,32 +50,32 @@ public function get_method(): string {
 	/**
 	 * Sets the start date for the payment activity data.
 	 *
-	 * @param string|null $date_start The start date in the format 'YYYY-MM-DDT00:00:00' or null.
+	 * @param string $date_start The start date in the format 'YYYY-MM-DDT00:00:00' or null.
 	 * @return void
 	 */
-	public function set_date_start( ?string $date_start ) {
-		// TBD - validation.
+	public function set_date_start( string $date_start ) {
+		$this->validate_date( $date_start, 'Y-m-d\TH:i:s' );
```

Comment:
Just like to point out that Server can accept any value parsable by [strtotime()](https://www.php.net/manual/en/function.strtotime.php).

I'm OK if we're restrictive here though.

Reply:
I think its OK to be restrictive in the public-facing API.